certified-connectors/Docurain/apiDefinition.swagger.json

```diff
@@ -0,0 +1,248 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "Docurain",
+    "description": "DocurainAPI Communication Connector. Docurain API allows you to create your own documents using template files registered in Docurain and data from PowerAppsn.",
+    "contact": {
+      "name": "Docurain support",
+      "url": "https://api.docurain.jp/console/inquiry",
+      "email": "cs@docurain.jp"
+    }
+  },
+  "host": "powerapps.docurain.jp",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/apiw/{outType}/{templateName}": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
```

Comment:
@geckoR42 Please replace the "default" responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.

Reply:
@sriyen-msft Removed "default". Fixed.